### PR TITLE
adding vessel density collections on behalf of @AlessandroScremin

### PR DESCRIPTION
**Pull request by Schpidi**
please review @AlessandroScremin

### CODE REVIEW

**Review by AlessandroScremin — DISMISSED** [review]

**Comment by Schpidi** [comment]
@Dorothyrono are you the right person to ask to take a look and eventually merge this? thx

File: collections/vessel_density_all.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Configurations:
+  - href: "https://docs.sentinel-hub.com/api/latest/api/process/"
+    rel: "about"
+    type: "text/html"
+    title: "Details about running Evalscripts"
+  - href: ""
+    rel: "processing-expression"
+    type: "application/javascript"
+    title: "Evalscript to generate Vessel Density imagery"
+    sentinelhub:layer_name: ""
```
Comment (Dorothyrono) [inline collections/vessel_density_all.yaml:43]: This cant be empty. A link to a visualisation evalscript is needed.   Same to the rest of the YAML files

@Schpidi 
not sure whether you or Allessandro will create the script also,  

In anycase we can help create the script, but not very soon , maybe within the next week or two.

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/vessel_density_all.yaml (hunk 1 above)
Comment (Schpidi) [reply collections/vessel_density_all.yaml:43]: @AlessandroScremin can you please take care of that? thanks

Comment (Dorothyrono) [reply collections/vessel_density_all.yaml:43]: both href and layer_name 

The layer name is given by you. You have done it correctly. Thanks

Comment (Dorothyrono) [reply collections/vessel_density_all.yaml:43]: both href and layer_name 

The layer name is given by you. You have done it correctly. Thanks

**Review by Dorothyrono — APPROVED** [review]

**Comment by Dorothyrono** [comment]
Looks good .. I think we can merge

**Pull request merged**